src/envs/pybullet_robots.py

```diff
@@ -250,42 +275,26 @@ def get_joints(self) -> Sequence[float]:
         return joint_state
 
     def set_motors(self, action_arr: Array) -> None:
-        ee_delta = (action_arr[0], action_arr[1], action_arr[2])
-        f_value = action_arr[3]
-        ee_link_state = p.getLinkState(self._fetch_id,
-                                       self._ee_id,
-                                       physicsClientId=self._physics_client_id)
-        current_position = ee_link_state[4]
-        target_position = np.add(current_position, ee_delta).tolist()
-
-        # We assume that the robot is already close enough to the target
-        # position that IK will succeed with one call, so validate is False.
-        # Furthermore, updating the state of the robot during simulation, which
-        # validate=True would do, is risky and discouraged by PyBullet.
-        joint_values = inverse_kinematics(
```

Comment:
so we're now doing IK in option policy, but setJointMotorControl2 in step(), right?

Reply:
right